src/screens/SavedEventListScreen/NoSavedEvents.js

```diff
@@ -52,6 +54,9 @@ const styles = StyleSheet.create({
     flex: 3,
     marginHorizontal: 12
```

Comment:
Instead of a fixed `marginHorizontal: 12` we should use the `ContentPadding` component here.